### PR TITLE
ContainerHelper: Added tests

### PR DESCRIPTION
**Pull request by juniwalk**
I must say, working with PHPUnit seems so much easier to me.

Anyway here is a draft of a tests, let's see how it goes.

### CODE REVIEW

File: tests/KdybyTests/Console/Parameters.phpt [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+	public function testParametersCommand()
+	{
+		Debugger::$logDirectory = TEMP_DIR . '/log';
+		Tester\Helpers::purge(Debugger::$logDirectory);
+
+		/** @var Nette\DI\Container $container */
+		$container = $this->prepareConfigurator()->createContainer();
+
+		/** @var Kdyby\Console\Application $app */
+		$app = $container->getByType('Kdyby\Console\Application');
```
Comment (fprochazka) [inline tests/KdybyTests/Console/Parameters.phpt]: I don't think you need to compile the container and run the app to test that those three new methods work. Unit test would suffice perfectly.

Comment (juniwalk) [inline tests/KdybyTests/Console/Parameters.phpt]: So if I understand correctly you want me to test just the ContainerHelper with some sort of Container mock? (newbie to testing)

Comment (fprochazka) [inline tests/KdybyTests/Console/Parameters.phpt]: Exactly :)

Comment (fprochazka) [inline tests/KdybyTests/Console/Parameters.phpt]: You probably don't even have to mock the container, just use `new Nette\DI\Container()` and pass some arguments and the do asserts on the helper

**Comment by juniwalk** [comment]
Alright, is it better now? Shall I add test for `getByType` while I am at it?

File: tests/KdybyTests/Console/ContainerHelper.phpt [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+		Assert::type($container, $helper->getContainer());
+		Assert::same('container', $helper->getName());
+	}
+
+
+	public function testParameters()
+	{
+		Debugger::$logDirectory = TEMP_DIR . '/log';
+		Tester\Helpers::purge(Debugger::$logDirectory);
```
Comment (fprochazka) [inline tests/KdybyTests/Console/ContainerHelper.phpt]: what are theese two lines for?

Comment (juniwalk) [inline tests/KdybyTests/Console/ContainerHelper.phpt]: Beats me, I had some weird errors before I rewrote that test, I will try to remove them.

**Comment by juniwalk** [comment]
Okay it works now, was probably because I was building container before.

**Comment by fprochazka** [comment]
Cool, thanks :+1:

**Pull request merged**